content/en/docs/ops/diagnostic-tools/proxy-cmd/index.md

```diff
@@ -61,48 +61,9 @@ where the issue may lie.
 
 {{< text bash json >}}
 $ istioctl proxy-status details-v1-6dcc6fbb9d-wsjz4.default
---- Istiod Clusters
```

Comment:
I'm a little concerned here since the text above says we are retrieving a diff, while the output being changed goes from a diff to things matching. It seems that there should be a diff in the output or maybe the section is no longer correct.

Reply:
I think I noticed this in `1.6` also but was not sure whether it returns diffs. I haven't found any diffs during my testing. I'm I missing something?

Reply:
cc @esnible Do you know if this is supposed to show a diff as the docs suggest?

Reply:
Either the whole section needs to be removed (the title contains diff and the last sentence says to see the diffs but they are removed in this PR) or something is broken and we aren't seeing the diffs we should.

Reply:
Looks to me like there would only be a diff if the environment is set up to cause an update problem. If everything is working properly, there should be no diff.

So, I suggest leaving the Clusters part as it was, just update the Routes part to include the last update time.

Reply:
If Istio is operating correctly, it will output that Envoy and Istio match.
If Envoy is rejecting the config supplied by Istiod then the command will show the difference between what Envoy believes and what Istiod believes.  Often this gives an advanced troubleshooter an idea about what is wrong.

Reply:
Keeping diffs as it is. Thanks.